docs/BUILD.bazel

```diff
@@ -26,18 +215,21 @@ alias(
     name = "defs",
     actual = "//python:defs_bzl",
     deprecation = "Use //python:defs_bzl instead; targets under //docs are internal.",
```

Comment:
BTW, I'd be happy to remove these deprecated targets by now.

Reply:
SGTM, I'll send another PR